src/bundle/Resources/public/scss/ui/modules/universal-discovery/_tree.scss

```diff
@@ -4,20 +4,14 @@
     position: relative;
 
     .m-tree {
-        width: initial !important; // TODO: remove when Content Tree refactored
+        width: initial !important;
```

Comment:
I think the rule should be removed when content tree is refactored not only todo ;)